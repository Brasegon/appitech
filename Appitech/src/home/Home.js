import React from 'react';
import {
  StyleSheet,
  Text,
  View,
  Image,
  StatusBar,
} from 'react-native';
import LinearGradient from 'react-native-linear-gradient';
import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';

  
export default class Home extends React.Component {     
  render() {
    return (
      <View style={styles.container}>

              <View style={{ backgroundColor: '#bbe1fa' }}>
        <View style={styles.container2}>
        <Image source={require('../../Asset/Appitech.png')} style={{ width: 230, height: 60, top:20, left: 80 }} />

            </View>
            </View>



        <View style={{ backgroundColor: '#0f4c75' }}>
                  <LinearGradient colors={['#bbe1fa', '#3282b8']} style={styles.container1}>
                <View style={{top:-25}}>
                    <Text style={styles.title}>Prochaine activité</Text>
                    <View style={{backgroundColor:"#F2F1F7",
                                  width:350, left:30, top:60, borderBottomLeftRadius: 30,
                                  borderBottomRightRadius: 30,
                                  borderTopRightRadius: 30,
            borderTopLeftRadius: 30, height: 100}}>
      <MaterialCommunityIcons name="calendar-blank-multiple" color={"#0f4c75"} size={50} style={{position:"absolute", top:20, left:20}} /> 
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>Review projet de fin d'année</Text>
                    <Text style={{fontSize:13, left:90, top:15, width:250, color:'grey'}}>Salle Scifi <MaterialCommunityIcons name="sign-direction" color={"grey"} size={15} /></Text>
                    <Text style={{fontSize:13, left:90, top:15, width:250 , color:'grey'}}>11h30 <MaterialCommunityIcons name="clock-time-eight-outline" color={"grey"} size={15} /></Text>

                    </View>
                    </View>
                  </LinearGradient>
        </View>

    <View style={{ backgroundColor: '#231332' }}>
        <View style={styles.container3}>
            <View style={{top:-30}}>
        <Text style={styles.title}>Projets en cours</Text>
        <View style={{backgroundColor:"#F2F1F7",
                                  width:350, left:30, top:60, borderBottomLeftRadius: 30,
                                  borderBottomRightRadius: 30,
                                  borderTopRightRadius: 30,
            borderTopLeftRadius: 30, height: 140}}>
      <MaterialCommunityIcons name="folder-open-outline" color={"#0f4c75"} size={50} style={{position:"absolute", top:20, left:20}} /> 
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>Epicture</Text>
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>E-Commerce</Text>
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>My_FTP</Text>
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>Minishell2</Text>

                    </View>
                    </View>
                    <View style={{marginTop:20}}>
                    <Text style={styles.title}>Dernières notes</Text>
        <View style={{backgroundColor:"#F2F1F7",
                                  width:350, left:30, top:60, borderBottomLeftRadius: 30,
                                  borderBottomRightRadius: 30,
                                  borderTopRightRadius: 30,
            borderTopLeftRadius: 30, height: 90}}>
      <MaterialCommunityIcons name="checkbox-marked-outline" color={"#0f4c75"} size={50} style={{position:"absolute", top:20, left:20}} /> 
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>Devops : <Text style={{color : "#0f4c75"}}>A</Text></Text>
                    <Text style={{fontSize:17, left:90, top:15, width:250, paddingBottom:5}}>Java : <Text style={{color : "#0f4c75"}}>B</Text></Text>
                    </View>
                    </View>

        </View>
    </View>


      </View>
    );
  }
}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
  container1: {
    height: 230,
    padding: 10,
    borderBottomLeftRadius:100,
  },
  container2: {
    backgroundColor: 'white',
    height: 130,
    padding: 10,
    borderBottomLeftRadius:100,
  },
  container3: {
      backgroundColor : '#0f4c75',
    height: 600,
    padding: 10,
    borderBottomLeftRadius:100,
  },
  title: {
      fontSize: 20,
      color : 'white',
      left : 50,
      top : 45

  }
});