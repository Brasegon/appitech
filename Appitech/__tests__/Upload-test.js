/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Home from '../src/home/Home';
 
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer';
 import Upload from '../src/home/Upload/Upload';
 
 it('Render Upload', () => {
   const tree = renderer.create(<Upload/>).toJSON();
   expect(tree).toMatchSnapshot();
 })