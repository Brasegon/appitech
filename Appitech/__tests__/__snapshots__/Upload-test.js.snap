// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Upload 1`] = `
<View>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    nativeBackgroundAndroid={
      Object {
        "attribute": "selectableItemBackground",
        "rippleRadius": undefined,
        "type": "ThemeAttrAndroid",
      }
    }
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "#2196F3",
          "borderRadius": 2,
          "elevation": 4,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontWeight": "500",
            "margin": 8,
            "textAlign": "center",
          },
        ]
      }
    >
      CHOISIR UNE IMAGE
    </Text>
  </View>
</View>
`;
